### PR TITLE
Review V1

### PR DESCRIPTION
**Pull request by thomasmcinnes**
I am a little stuck. I don't really know much about the import. A lot of new stuff. Please, could you tell me if I am going wrong with?
not sure if reading the import or if the FOR and IF function.

### CODE REVIEW

**Comment by thomasmcinnes** [comment]
Review V1 
I am a little suck. I don't really know much about the import. A lot of new stuff. Please, could you tell me where I am going wrong?
not sure if it is reading the import or if the FOR and IF function.

File: Statements.java [hunk 1]
```diff
@@ -1,46 +1,49 @@
...
-        float fred;
-        double donna;
-        char colin;
-        boolean bob = true;
-        isPrime(irene);
+    private static final Object Prime1000 = getPrime1000;
+
+    public static void main(String[] args)throws Exception
+    {
+        //int irene = 0;
```
Comment (neveronline79) [inline Statements.java:10]: Try not to leave lines in code that are commented out

File: Statements.java [hunk 2]
```diff
@@ -1,46 +1,49 @@
...
+        //isPrime(irene);
 
         int primeCount = 0;
+        File Prime1000 = new getPrime1000("C:\\Users\\thomas.mcinnes\\Documents\\Prime numbers\\Prime1000.txt");
+
```
Comment (neveronline79) [inline Statements.java:15]: FileReader fileReader = new FileReader(...

File: Statements.java [hunk 3]
```diff
@@ -1,46 +1,49 @@
...
-        isPrime(irene);
+    private static final Object Prime1000 = getPrime1000;
+
+    public static void main(String[] args)throws Exception
+    {
+        //int irene = 0;
+        //isPrime(irene);
 
         int primeCount = 0;
+        File Prime1000 = new getPrime1000("C:\\Users\\thomas.mcinnes\\Documents\\Prime numbers\\Prime1000.txt");
```
Comment (neveronline79) [inline Statements.java:14]: I think you might want something along the lines of:

File f = new File("C:\\Users\\thomas.mcinnes\\Documents\\Prime numbers\\Prime1000.txt")

**Review by neveronline79 — CHANGES_REQUESTED** [review]
When this works after making the changes, let me know and I can re-review

**Pull request closed without merging**